index 1803a42e..b1b21465 100644
--- a/src/Corrade/Utility/CMakeLists.txt
+++ b/src/Corrade/Utility/CMakeLists.txt
@@ -272,7 +272,6 @@ endif()
 # However, it seems like this feature never materialized, as doing this will
 # result in corrade-rc that's looking for vcruntime140_app.dll in order to be
 # run. Last checked: Nov 2019.
-if(NOT CMAKE_CROSSCOMPILING)
     # Sources for standalone corrade-rc
     set(CorradeUtilityRc_SRCS
         Arguments.cpp
@@ -333,6 +332,7 @@ if(NOT CMAKE_CROSSCOMPILING)
     endif()
     install(TARGETS corrade-rc DESTINATION ${CORRADE_BINARY_INSTALL_DIR})
 
+if(NOT CMAKE_CROSSCOMPILING)
     # Corrade::rc target alias for superprojects
     add_executable(Corrade::rc ALIAS corrade-rc)
 endif()
